auth0-js/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Adrian Chia <https://github.com/adrianchia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace auth0 {
+declare module auth0 {
```

Comment:
This compiles fine but failed at run time when running through Webpack saying that "auth0 is not defined". Similar issue at https://github.com/s-panferov/awesome-typescript-loader/issues/246